README.md

```diff
@@ -52,26 +52,26 @@ gulp.task('templates', function() {
 
 All options supported by the [Jade API](http://jade-lang.com/api/) are supported
```

Comment:
Nah, I meant to change it to:
```
[Pug API](http://jade-lang.com/api/)
```
Then I can change the URL in the future when it changes.  But it should also redirect when it updates anyways.

Then I'll go ahead and let @phated merge when he wants.